tools/js-tools/eslintrc/tests.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * This is a workaround for a feature not available in ESLint, yet.
+ *
+ * @see https://github.com/eslint/eslint/issues/3458
+ * @todo Remove this when the above feature is natively available in ESLint
+ */
+require( '@rushstack/eslint-patch/modern-module-resolution' );
+
+module.exports = {
+	parser: '@babel/eslint-parser',
+	extends: [ 'plugin:jest/recommended' ],
+	plugins: [ 'jest' ],
+	env: {
+		browser: true,
+		es6: true,
+		mocha: true,
+		node: true,
+		jquery: true,
+		jest: true,
+	},
+	parserOptions: {
+		ecmaVersion: 2019,
+		ecmaFeatures: {
+			jsx: true,
+		},
+		requireConfigFile: false,
+	},
+	settings: {},
```

Comment:
How much of this is actually necessary? Much of it seems redundant to what's in base.js, which should probably be extended alongside of this.

Going by what's being changed in the one file using this file, all that would be needed here is
```suggestion
	extends: [ 'plugin:jest/recommended' ],
```
although based on the other 16 files using `plugin:jest/recommended` you could probably move a few more bits out of that file and into this one.
```suggestion
	extends: [ 'plugin:jest/recommended' ],
	env: { jest: true },
	rules: {
		'jsdoc/check-tag-names': [
			1, // Recommended
			{ definedTags: [ 'jest-environment' ] },
		],
	},
```

Reply:
Good suggestion. Thank you.